actionpack/lib/abstract_controller/translation.rb

```diff
@@ -20,7 +26,9 @@ def translate(key, **options)
         options[:default] = defaults.flatten
         key = "#{path}.#{action_name}#{key}"
       end
-      I18n.translate(key, **options)
+
+      i18n_raise = options.fetch(:raise, ActionController::Base.raise_on_missing_translations)
+      I18n.translate(key, options.merge!(raise: i18n_raise))
```

Comment:
These options need to be splatted to prevent a warning on Ruby 2.7:

https://buildkite.com/rails/rails/builds/69325#216048d0-21ef-469c-84d4-8e967af55241

```suggestion
      I18n.translate(key, **options, raise: i18n_raise)
```